pages/k8s/1.28/release-notes.md

```diff
@@ -0,0 +1,88 @@
+
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "1.28 Release notes"
+  description: Release notes for Charmed Kubernetes
+keywords: kubernetes, release, notes
+tags: [news]
+sidebar: k8smain-sidebar
+permalink: 1.28/release-notes.html
+layout: [base, ubuntu-com]
+toc: False
+---
+# 1.28
+
+### August 16, 2023 - `charmed-kubernetes --channel 1.28/stable`
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.28/bundle.yaml).
+
+## What's new
+
+### Broader adoption of ops charming
+
+We're pleased to announce a number charms have been refactored from the `reactive` and `pod-spec` styles to `ops` style.  These uplifts were taken in an effort to have access to charm libraries, better support from juju, and better community support when demonstrating charming.  Some of the uplifted charms include:
+
+* calico
+* metallb
+
+And coming soon are a more charms like
+* kubeapi-load-balancer
+* kubernetes-worker
+* kubernetes-control-plane
+
+### Ceph CSI adds CephFS
+
+Providing much needed feature parity to the aging cdk-addons, the ceph-csi charm now supports operations of ceph-fs along with the already supported ceph-rdb.
+
+## Component Versions
+
+### Charm(/Addons) pinned versions
+- kube-ovn 1.11.10
+- calico 3.25.1
+- ceph-csi 3.7.2 / 3.7.2
+- cinder-csi 1.27.1 / 1.26.2 
+- coredns 1.10.1 / 1.9.4
+- ingress-nginx 1.6.4
+- k8s-keystone-auth 1.26.2
+- kube-state-metrics 2.0.0 / 2.8.2
+- kubernetes-dashboard 2.7.0 / 2.7.0
+- openstack-cloud-controller-manager 1.27.1 / 1.26.2 
+
+### Charm default versions
+- cloud-provider-vsphere 1.26
+- vsphere-csi-driver 3.0.0
+- cloud-provider-azure 1.25.0
+- azuredisk-csi-driver 1.23.0
+- cloud-provider-aws 1.26.1
+- aws-ebs-csi-driver 1.12.0
+- gcp-compute-persistent-disk-csi-driver 1.8.0
+
+
+## Fixes
+
+A list of bug fixes and other minor feature updates in this release can be found at
+[the launchpad milestone page for 1.28](https://launchpad.net/charmed-kubernetes/+milestone/1.28).
+
+
+## Notes and Known Issues
+
```

Comment:
@charmed-kubernetes/canonical-k8s-team what would you like to call out here as risky spots?